clients/src/main/resources/common/message/ListOffsetsRequest.json

```diff
@@ -34,9 +34,12 @@
   // Version 7 enables listing offsets by max timestamp (KIP-734).
   //
   // Version 8 enables listing offsets by local log start offset (KIP-405).
-  "validVersions": "0-8",
+  //
+  // Version 9 enables listing offsets by last tiered offset (KIP-1005).
+  "validVersions": "0-9",
   "deprecatedVersions": "0",
   "flexibleVersions": "6+",
+  "latestVersionUnstable": true,
```

Comment:
This caused quite a few test failures in ListOffsetsRequestTest. Since the latest version of ListOffsetsRequest is unstable, the broker throws InvalidRequestException when the client sends the latest version. ListOffsetsRequestTest wasn't designed to test enableUnstableLastVersion, which could be fixed. However, ListOffsetsRequest is also used for inter broker communications. I am not sure how easy it is to support using an unstable version of the request for inter broker communication.

`org.apache.kafka.common.errors.InvalidRequestException: Received request api key LIST_OFFSETS with version 9 which is not enabled`

The easiest approach is to revert the changes in ListOffset for v9 in this PR and add it back when it's considered stable. @clolov : What do you think?

Reply:
I believe if interbroker communication is set up to only advertise stable versions (unless we allow them) and we use api versions to check the supported versions of the request, this should work for interbroker requests. I'm not sure the work to implement that, but I think it may end up being easier to revert the v9 version and solve in a follow up.

Reply:
The test just requires setting `unstable.api.versions.enable`, which I will do. We want to test unstable versions in JUnit tests (but not to expose them to users, until they're stable)